### PR TITLE
fixed responsiveness in hero section #234

### PR DESCRIPTION
**Pull request by ninman27**
changed background colors, and pictures, and improved responsiveness in the hero section...

*** I think @kire21 should change the reusable button component for the mobile version...

dark-desktop-hero-section

dark-mobile-hero-section

light-desktop-hero-section

light-mobile-hero-section

### CODE REVIEW

**Comment by edichoska** [comment]
Screenshot 2024-05-10 at 13 47 03

![Uploading Screenshot 2024-05-10 at 13.47.39.png…](

This still doesn't look good.

**Comment by edichoska** [comment]
Screenshot 2024-05-10 at 13 47 39

**Comment by edichoska** [comment]
@ninman27 can you do this instead " change the reusable button component for the mobile version"?

**Comment by ninman27** [comment]
I made changes, please check!

File: components/module-components/hero/Hero.tsx [hunk 1]
```diff
@@ -1,37 +1,60 @@
...
+              icon={<i className="bi bi-arrow-right" />}
+            />
+          </Link>
+        </div>
+        <div className={style.heroPictureContainer}>
+          <Image
+            src={lightTheme ? LightHeroImageDesktop : DarkHeroImageDesktop}
+            className={`${style.heroImage} ${style.desktop}`}
+            alt="hero picture desktop"
+            priority
```
Comment (edichoska) [inline components/module-components/hero/Hero.tsx:47]: great use of the priority property ⭐

**Pull request merged**